common/djangoapps/embargo/forms.py

```diff
@@ -11,15 +11,11 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
-from embargo.models import (
-    EmbargoedCourse, EmbargoedState, IPFilter,
-    RestrictedCourse
-)
-from embargo.fixtures.country_codes import COUNTRY_CODES
+from embargo.models import IPFilter, RestrictedCourse
 
 
-class CourseKeyValidationForm(forms.ModelForm):
```

Comment:
This was a form I created when implementing the country access feature; I can't find it being used anywhere else.